### PR TITLE
Lock wrangler behind enterprise

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
At John's request, locking wrangler behind a license. This change will make it so that any license (Professional, Enterprise, E10/E20, etc) will allow the wrangler feature to be used. 

#### Release Note

```release-note
Move Threads (Experimental) requires a Professional/Enterprise license to activate
```

### CODE REVIEW

File: webapp/channels/src/components/admin_console/admin_definition.tsx [hunk 1]
```diff
@@ -3203,7 +3203,7 @@ const AdminDefinition: AdminDefinitionType = {
                 url: 'site_config/wrangler',
                 title: t('admin.sidebar.move_thread'),
                 title_default: 'Move Thread (Beta)',
-                isHidden: it.any(it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.POSTS)), it.configIsFalse('FeatureFlags', 'MoveThreadsEnabled')),
+                isHidden: it.any(it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.POSTS)), it.configIsFalse('FeatureFlags', 'MoveThreadsEnabled'), it.not(it.any(it.licensedForSku('enterprise'), it.licensedForSku('e20')))),
```
Comment (nickmisasi) [inline webapp/channels/src/components/admin_console/admin_definition.tsx:3206]: So many double negatives 

If any of the following are true, the menu item is hidden:
- The user DOES NOT have permission
- The feature flag IS FALSE
- If all of the following are NOT TRUE
-- The license is enterprise
-- The license is e20

Comment (johndavidlugtu) [reply webapp/channels/src/components/admin_console/admin_definition.tsx:3206]: Should also add Professional ([reference](https://docs.google.com/document/d/1UCaQkeahlSa0Uv_TiGB7D3LK1vBCEsfvjkU6cV5xeZE/edit#heading=h.pcx78kd0udng))

Comment (nickmisasi) [reply webapp/channels/src/components/admin_console/admin_definition.tsx:3206]: In that case, I've updated the PR to simply enforce the existence of a license. Pro/Ent/E10/E20 will all be covered by this

Comment (johndavidlugtu) [reply webapp/channels/src/components/admin_console/admin_definition.tsx]: Sounds good

**Review by johndavidlugtu — APPROVED** [review]
Thanks Nick

**Review by gabrieljackson — APPROVED** [review]
LGTM

**Review by hmhealey — APPROVED** [review]

**Comment by hmhealey** [comment]
To echo some discussion we've had here in case anyone comes across this PR and is concerned about this being EE-only, we're currently planning to make most of this available without a license once it's no longerexperiemental. It's just easier for us to work on it and break things while there are fewer people depending on the current functionality which might change

**Pull request merged**